setup.sh

```diff
@@ -1,10 +1,83 @@
 #!/usr/bin/env bash
 
-export SERVER_ADDRESS="$1"
-export HULY_SECRET="secret"
+# Define the default values
+HULY_VERSION="v0.6.295"
+DOCKER_NAME="huly"
+CONFIG_FILE="huly.conf"
 
-echo "Setting Huly Server Address: $SERVER_ADDRESS"
-envsubst < template.conf > nginx.conf
-envsubst < template.env > .env
 
-./use-version.sh v0.6.295
+if [ -f "$CONFIG_FILE" ]; then
+    source "$CONFIG_FILE"
+fi
+clear
+
+while true; do
+    read -p "Enter the host address (domain name or IP) [default: ${HOST_ADDRESS:-localhost}]: " input
+    _HOST_ADDRESS="${input:-${HOST_ADDRESS:-localhost}}"
+    #TODO: proper validation
+    #if [[ "$_HOST_ADDRESS" =~ ^[a-zA-Z0-9.-]+$ || "$_HOST_ADDRESS" =~ ^([0-9]{1,3}\.){3}[0-9]{1,3}$ ]]; then
+        break
+    #else
+    #   echo "Invalid host address. Please enter a valid domain name or IP."
+    #fi
+done
+clear
```

Comment:
Please don't clear, it confuses when values selected on previous steps disappear.

Reply:
Instead of clearing after each step, I’ll clear once when all inputs have been completed. 
I will provide a summary of the selected values after clearing.